File: backend/member-service/src/main/java/com/example/memberservice/config/ModelMapperConfig.java
package com.example.memberservice.config;

import lombok.Setter;
import org.modelmapper.ModelMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.transaction.annotation.EnableTransactionManagement;
import org.springframework.validation.annotation.Validated;

@Setter
@Validated
@Configuration
@EnableTransactionManagement
public class ModelMapperConfig {

    @Bean
    public ModelMapper modelMapper() {
        return new ModelMapper();
    }
}

File: backend/member-service/src/main/kotlin/com/example/memberservice/config/ModelMapperConfig.kt
package com.example.memberservice.config

import org.modelmapper.ModelMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.transaction.annotation.EnableTransactionManagement

@Configuration
@EnableTransactionManagement
class ModelMapperConfig {

    @Bean
    fun modelMapper(): ModelMapper = ModelMapper()
}
